src/ZGUtilities.vala

```diff
@@ -101,20 +119,4 @@ namespace SecurityPrivacy.Utilities {
 
         return blacklist;
     }
-
-    public static Variant to_variant (HashTable<string, Zeitgeist.Event> blacklist) {
```

Comment:
I see this seems to use `to_variant ()`:
https://github.com/elementary/switchboard-plug-security-privacy/blob/f8928f5f47b85322c64d43c704ddbfdede5014de/src/BlackList.vala#L79

but everything seems to work on this branch even with this method removed.